payment-request/payment-request-constructor.https.html

```diff
@@ -132,10 +194,13 @@
 }, 'Rethrow any exceptions of JSON-serializing paymentMethod.data into ' +
    'a string');
 
-const invalidMonetaryAmounts = ['-', 'notdigits', 'ALSONOTDIGITS', '10.', '.99',
-  '-10.', '-.99', '10-', '1-0', '1.0.0', '1/3', '', null,
+// process total
+const invalidAmounts = ['-', 'notdigits', 'ALSONOTDIGITS', '10.', '.99',
+  '-10.', '-.99', '10-', '1-0', '1.0.0', '1/3', '', null, ' 1.0  ', ' 1.0 ', 
+  '1.0 ', 'USD$1.0', '$1.0', {toString() { return " 1.0"}} 
```

Comment:
There’s a space at the end of each of the above two lines that needs to be removed to make the linter happy